sweagent/agent/models.py

```diff
@@ -225,6 +225,11 @@ class OpenAIModel(BaseModel):
             "cost_per_input_token": 5e-06,
             "cost_per_output_token": 15e-06,
         },
+        "gpt-4o-mini": {
```

Comment:
Could you name this `gpt-4o-mini-2024-07-18` and then if you want, put `gpt-4o-mini` in the `SHORTCUTS` pointing to `gpt-4o-mini-2024-07-18`?